data/starts.txt

```diff
@@ -46,8 +46,9 @@ conversation "default intro"
 
 start "hai space after FW and HR"
 	name "Advanced Hai Start"
-	description `You grew up on Greenwater, a world with a very rich and diverse hai-human culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard in a texture farm making human clothing that, quite surprisingly, sold very well.`
-	description `	You heard stories of heroes saving both humans and Hai. Now, as you enter the shipyard, you feel like this is the start of a new life.`
+	description `You grew up on Greenwater, a world with a very rich and diverse hai-human culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard in a texture farm making human clothing that sold very well.`
+	description `	You heard stories of heroes saving both humans and Hai.`
```

Comment:
Okay, now we just need a reason why this fact is mentioned. I know why _you_ want it there, but it doesn't really make sense for FL; you first talk about FL's life, then this random story bit, then back to what FL's doing rn. At least now it doesn't imply that FL and these events are somehow connected, so that's fixed.

Reply:
its mentionned because thats part of why you take to the stars
```suggestion
	description `	You were inspired by stories of heroes saving both humans and Hai.`
```